README.md

```diff
@@ -1,57 +1,58 @@
-# A template repo for IIASA Python projects
+# Documentation of the teaching learning material by the Energy, Climate, and Environment Program (ECE)
 
 Copyright (c) 2021 IIASA
 
-![License](https://img.shields.io/github/license/iiasa/python-stub)
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+![License](https://img.shields.io/github/license/iiasa/ece-teaching)
 
-## Overview
+This is a [Sphinx](http://sphinx-doc.org/) project for the documentation of the build of the teaching and learning website by the Energy, Climate, and Environment Program (ECE). The website can be found [here](https://teaching.ece.iiasa.ac.at/).
 
-Template repository for creating python packages and Sphinx-based documentation pages in line with the IIASA design guidelines
+## Scope & Aim
 
-## Configuration
+The scope and aim of the repository is to create a Sphinx documentation, which is the basis for the teaching and learning website. This repository should contain all docs, but not the data, of the teaching and learning material. 
```

Comment:
The repository does not document the build, it *is* the build - better to use more direct language. Also, please use hard line-breaks at 88 characters.

Probably also good to have a short section here (before the technical instructions) about what kind of material is included in "learning and teaching" - workshops organized by ECE, university courses taught by ECE staff, tutorials, etc.

Third comment, the difference between docs and data is very subtle and might be easy to grasp for all potential contributors.

Maybe better:
> This repo contains pages with relevant information about teaching events, including links to relevant websites and the teaching material (e.g., pdf, pptx, recorded videos)
> Any such material should not be committed to this repository, but be stored in an appropriate location
> such as the IIASA publication repository (pure.iiasa.ac.at) or a user folder on ldom002.

Reply:
Thanks for your review!
Just to be sure regarding the hard line-breaks: The width of the text on the page should not move if the screen gets extended but rather stay at max 88 characters. E.g.:
```
This repo contains pages with relevant information about teaching events, including <br/>
links to relevant websites and the teaching material (e.g., pdf, pptx, recorded videos). <br/>
Any such material should not be committed to this repository, but be stored in an <br/>
appropriate location, such as the IIASA publication repository (pure.iiasa.ac.at) or a <br/>
user folder on ldom002.
```

Reply:
Didn't mean "no hard breaks in the rendered page" (not `<br/>`), just have line breaks in the rst file.